File: Programming Basics/Conditional Statements Advanced - Exercises/05.Journey.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        double budget = double.Parse(Console.ReadLine());
        string season = Console.ReadLine();

        switch (season)
        {
            case "summer":
                if (budget <= 100)
                {
                    Console.WriteLine("Somewhere in Bulgaria");
                    Console.WriteLine($"Camp - {budget -= (budget * 0.70):f2}");
                }
                else if (budget <= 1000)
                {
                    Console.WriteLine("Somewhere in Balkans");
                    Console.WriteLine($"Camp - {budget -= (budget * 0.60):f2}");
                }
                else
                {
                    Console.WriteLine("Somewhere in Europe");
                    Console.WriteLine($"Hotel - {budget -= (budget * 0.10):f2}");
                }
                break;
            case "winter":
                if (budget <= 100)
                {
                    Console.WriteLine("Somewhere in Bulgaria");
                    Console.WriteLine($"Hotel - {budget -= (budget * 0.30):f2}");
                }
                else if (budget <= 1000)
                {
                    Console.WriteLine("Somewhere in Balkans");
                    Console.WriteLine($"Hotel - {budget -= (budget * 0.20):f2}");
                }
                else
                {
                    Console.WriteLine("Somewhere in Europe");
                    Console.WriteLine($"Hotel - {budget -= (budget * 0.10):f2}");
                }
                break;
        }
    }
}

File: Programming Basics - Python/Conditional Statements Advanced - Exercises/05.Journey.py
budget = float(input())
season = input()

type = ''
destination = ''

if budget <= 100:
    destination = 'Bulgaria'
    if season == 'summer':
        budget *= 0.30
        type = 'Camp'
    elif season == 'winter':
        budget *= 0.70
        type = 'Hotel'

elif 100 < budget <= 1000:
    destination = 'Balkans'
    if season == 'summer':
        budget *= 0.40
        type = 'Camp'
    elif season == 'winter':
        budget *= 0.80
        type = 'Hotel'

elif budget > 1000:
    destination = 'Europe'
    type = 'Hotel'
    budget *= 0.90

print(f'Somewhere in {destination}')
print(f'{type} - {budget:.2f}')